### ISSUE TITLE
[Feature] HD-trailers Reloaded button in VideoInfoDialog

### ISSUE THREAD

**Issue by Mr-7mdan** [jurialmunkey/skin.arctic.horizon.2#764]
### Skin section

Video Info Dialog

### Feature description

It would be really helpful to add a button to view HQ trailers using this new add-on

https://forum.kodi.tv/showthread.php?tid=366357

### Why is this feature needed? Please justify and explain.

Add an option too view trailers in high quality instead of the low quality YouTube option

### Checklist

- [X] I have made a separate issue for each feature I am requesting.
- [X] I have searched the issues and this feature has not been requested.
- [X] I am using the most recent skin version.

**Comment by psp2111-ADSLGATE** [comment]
if want more than 720p resolution from YouTube you need to enable "MPEG-DASH"

Go to "Kodi add-on repository" and install "YouTube addon" from Video Add-ons and "InputStream Adaptive" from VideoPlayer InputStream.
after that go to YouTube addon Settings and enable "use MPEG-DASH" and change video quality to whatever you want.
1080p-2160p Trailers should work now.

Note: you don't need to add your own API Keys for YouTube if you just want to play the trailers.

**Comment by Mr-7mdan** [comment]
> if want more than 720p resolution from YouTube you need to enable "MPEG-DASH"
> 
> Go to "Kodi add-on repository" and install "YouTube addon" from Video Add-ons and "InputStream Adaptive" from VideoPlayer InputStream. after that go to YouTube addon Settings and enable "use MPEG-DASH" and change video quality to whatever you want. 1080p-2160p Trailers should work now.
> 
> Note: you don't need to add your own API Keys for YouTube if you just want to play the trailers.

Thanks @psp2111-ADSLGATE for the response. Unfortunately, I was unable to get MPEG-DASH enabled. All dependencies are met, yet it is disabled for some reason.

![image](https://user-images.githubusercontent.com/9439671/213906021-1601304e-3d11-4ec8-a34c-41fab774c03c.png)

**Comment by psp2111-ADSLGATE** [comment]
You have to install "InputStream Adaptive" addon to use MPEG-DASH.

go to "Addons" -> "Kodi add-on repository" -> "VideoPlayer InputStream" then install InputStream Adaptive

**Comment by Mr-7mdan** [comment]
> You have to install "InputStream Adaptive" addon to use MPEG-DASH.
> 
> go to "Addons" -> "Kodi add-on repository" -> "VideoPlayer InputStream" then install InputStream Adaptive

Thank you @psp2111-ADSLGATE  the MPEG-DASH worked and now I am getting HQ Trailers from YouTube. 
BTW, I was able to add a button for HD-Trailers and run a python code to scrap the site and prompt the user to select desired quality and then play the trailer from HD-Trailers website. But it's always a hassle for non-skinners to change xml files especially when it as sophisticated as AH2. I always wished for an option to add a custom user button in the setting and link it to py script. It won't hurt.
I did my own script to modify the xml files every time I update AH2. But this too can break if a significant change was made to the skin.
